### PR TITLE
Put side conditions on p1 and p2 in multi battles

### PR DESCRIPTION
**Pull request by MacChaeger**

### CODE REVIEW

**Comment by MacChaeger** [comment]
Makes #5341 unnecessary.

**Comment by Zarel** [comment]
I think you mean #5340? The concept of a PR making itself unnecessary is a rather more existential question than what you probably actually meant. ;)

**Comment by MacChaeger** [comment]
;(

File: data/mods/gen3/moves.js [hunk 1]
```diff
@@ -301,10 +301,7 @@ let BattleMovedex = {
 	doomdesire: {
 		inherit: true,
 		onTry(source, target) {
-			target.side.addSideCondition('futuremove');
-			if (target.side.sideConditions['futuremove'].positions[target.position]) {
-				return false;
-			}
+			target.side.addSlotCondition(target, 'futuremove');
```
Comment (Slayer95) [inline data/mods/gen3/moves.js]: ``if (!...) return false;`` ???

File: data/mods/gen4/moves.js [hunk 2]
```diff
@@ -431,9 +431,6 @@ let BattleMovedex = {
 		desc: "Deals typeless damage that cannot be a critical hit two turns after this move is used. Damage is calculated against the target on use, and at the end of the final turn that damage is dealt to the Pokemon at the position the original target had at the time. Fails if this move or Future Sight is already in effect for the target's position.",
 		onTry(source, target) {
 			target.side.addSideCondition('futuremove');
```
Comment (Slayer95) [inline data/mods/gen4/moves.js]: ``slotCondition``?

**Comment by Zarel** [comment]
Looks good to me.

**Comment by Zarel** [comment]
Will merge in a day if there are no further updates.

**Comment by Zarel** [comment]
Sorry, mistaken close; got this confused with #5340

**Pull request merged**